tests/scripts/action.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+
+set -x
+
+if [$# -eq 3]; then
+    DAEMON=$2
+    OSD_COUNT=$3
+fi
+
+function install_deps() {
+    sudo wget https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_amd64 -O /usr/bin/yq
+    sudo chmod +x /usr/bin/yq
+}
+
+function check_k8s_cluster_status() {
```

Comment:
```suggestion
function print_k8s_cluster_status() {
```

Reply:
I have changed the action name accordingly so it is easier to look for.